### DISCUSSION TITLE
Text disappears off bottom of page when inserting vertical space

### DISCUSSION THREAD

**Discussion by ghost** [xournalpp/xournalpp#5128] [Q&A] [answered]
Hi,
I am currently using this program to take digital handwritten notes. When I want to insert some information in the middle of an existing page of notes, I use the vertical space tool to move some of the existing writing downward, freeing up a blank space for me to write in. However, if there is no blank space at the bottom of the page, this pushes some of my writing off the bottom of the page, where it disappears from view. Are there any settings I can change so the text automatically moves onto the next page instead of disappearing between pages?

**Comment by rolandlo** [comment, marked as answer]
No, this feature is currently not implemented. It would be nice to have, I agree.
